Client/Frontend/Settings/Brave Search/BraveSearchDebugMenuDetail.swift

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import BraveUI
+
+private let log = Logger.browserLogger
+
+struct BraveSearchDebugMenuDetail: View {
+  let logEntry: BraveSearchLogEntry.FallbackLogEntry
+  
+  @State private var showingSheet = false
+  
+  var body: some View {
+    List {
+      HStack {
+        Text("URL")
+        Spacer()
+        Text(logEntry.url.absoluteString)
+      }
```

Comment:
All your inner view's here match this same structure, you're probably better off creating a small private view that does just this:

```swift
private struct ValueRow: View {
    var title: LocalizedStringKey
    var value: String
    
    var body: some View {
        HStack {
            Text(title)
            Spacer()
            Text(value)
        }
    }
}
```